libs/ui-lib/lib/common/components/ui/formik/validationSchemas.test.ts

```diff
@@ -0,0 +1,6 @@
+import { test } from 'vitest';
+import { noProxyValidationSchema } from './validationSchemas';
+
+test('noProxyValidationSchema runs', async () => {
+  await noProxyValidationSchema.validate('.y.com');
```

Comment:
What should anyone looking at this test understand about this test?
What's the expected value/result?
Under what circumstance the test should pass/fail?

Reply:
It's a stub, I have a pending PR actually implementing that particular test, it will follow right after we agree on the infra here. I was just lazy to do `Hello, world`.